app/controllers/settings/password.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    changePassword(passwordData) {
+      let payload = {
+        'data': {
+          'old-password' : passwordData.passwordCurrent,
+          'new-password' : passwordData.passwordNew
+        }
+      };
+      this.get('loader')
+        .post('/auth/change-password', payload)
+        .then(data => {
+          if (data.error) {
+            this.get('notify').error(this.l10n.t(data.error));
+          } else {
+            this.get('notify').success(this.l10n.t('Password updated successfully'));
+          }
+          this.set('passwordCurrent', '');
+          this.set('passwordNew', '');
+          this.set('passwordRepeat', '');
+        })
+        .catch(error => {
+          this.get('notify').error(this.l10n.t(error));
```

Comment:
@sumedh123 This returns the following message if the request fails. 
```
Could not make POST request to https://open-event-api.herokuapp.com/v1/auth/change-password
```
I think we should not expose the endpoint of the request, and follow the a unified approach as other requests something like:
```
An unexpected error has occurred. Password did not change.
```